x-pack/plugins/stack_alerts/server/rule_types/es_query/executor.ts

```diff
@@ -161,6 +167,7 @@ export async function executor(core: CoreSetup, options: ExecutorOptions<EsQuery
         [ALERT_TITLE]: actionContext.title,
         [ALERT_EVALUATION_CONDITIONS]: actionContext.conditions,
         [ALERT_EVALUATION_VALUE]: `${actionContext.value}`,
+        [ALERT_EVALUATION_THRESHOLD]: first(params.threshold),
```

Comment:
ES query rule threshold could be 2 numbers if the comparator is `between` or `not between` but the evaluation threshold is expecting a single value.